lib/resources/aws/aws_ebs_volume.rb

```diff
@@ -0,0 +1,123 @@
+class AwsEbsVolume < Inspec.resource(1)
+  name 'aws_ebs_volume'
+  desc 'Verifies settings for an EC2 volume'
+
+  example <<-EOX
+    describe aws_ebs_volume('i-123456') do
+      it { should be_encrypted }
+      its('size') { should cmp 8 }
+    end
+
+    describe aws_ebs_volume(name: 'my-volume') do
+      its('encrypted') { should eq true }
+      its('iops') { should cmp 100 }
+    end
+EOX
+  supports platform: 'aws'
+
+  # TODO: rewrite to avoid direct injection, match other resources, use AwsSingularResourceMixin
+  def initialize(opts, conn = nil)
+    @opts = opts
+    @opts.is_a?(Hash) ? @display_name = @opts[:name] : @display_name = opts
```

Comment:
Could we rewrite this to match the others? 

Example:
```
@display_name = opts.is_a?(Hash) ? @opts[:name] : opts
```

Reply:
Fixed